modules/io/include/visp3/io/vpImageIo.h

```diff
@@ -138,11 +147,11 @@ class VISP_EXPORT vpImageIo
   static void readPPM(vpImage<unsigned char> &I, const std::string &filename);
   static void readPPM(vpImage<vpRGBa> &I, const std::string &filename);
 
-  static void readJPEG(vpImage<unsigned char> &I, const std::string &filename);
-  static void readJPEG(vpImage<vpRGBa> &I, const std::string &filename);
+  static void readJPEG(vpImage<unsigned char> &I, const std::string &filename, int backend=IO_DEFAULT_BACKEND);
+  static void readJPEG(vpImage<vpRGBa> &I, const std::string &filename, int backend=IO_DEFAULT_BACKEND);
 
-  static void readPNG(vpImage<unsigned char> &I, const std::string &filename);
-  static void readPNG(vpImage<vpRGBa> &I, const std::string &filename);
+  static void readPNG(vpImage<unsigned char> &I, const std::string &filename, int backend=IO_DEFAULT_BACKEND);
+  static void readPNG(vpImage<vpRGBa> &I, const std::string &filename, int backend=IO_DEFAULT_BACKEND);
```

Comment:
Use `int` type for `backend` otherwise Java binding will not work and I don't know how to solve it.

Reply:
ok